### PR TITLE
dev-libs/libplatform: build shared library, EAPI=7

### PR DESCRIPTION
**Pull request by candrews**
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Craig Andrews <candrews@gentoo.org>

### CODE REVIEW

File: dev-libs/libplatform/libplatform-2.1.0.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Pulse-Eight/${MY_PN}/archive/${MY_PN_PREFIX}-${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${MY_PN}-${MY_PN_PREFIX}-${MY_PN}-${PV}"
+	KEYWORDS="~amd64 ~arm ~x86"
+fi
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=ON
```
Comment (a17r) [inline dev-libs/libplatform/libplatform-2.1.0.1-r1.ebuild]: already set by `cmake-utils.eclass` in `EAPI-7`

File: dev-libs/libplatform/libplatform-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Pulse-Eight/${MY_PN}/archive/${MY_PN_PREFIX}-${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${MY_PN}-${MY_PN_PREFIX}-${MY_PN}-${PV}"
+	KEYWORDS="~amd64 ~arm ~x86"
+fi
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=ON
```
Comment (a17r) [inline dev-libs/libplatform/libplatform-9999.ebuild]: already set by `cmake-utils.eclass` in `EAPI-7`

File: dev-libs/libplatform/libplatform-2.1.0.1-r1.ebuild (hunk 1 above)
Comment (candrews) [reply dev-libs/libplatform/libplatform-2.1.0.1-r1.ebuild]: Thanks, I didn't notice that change in cmake-utils.eclass.

**Comment by candrews** [comment]
Merged!

**Pull request closed without merging**